projects/next-dropzone/src/lib/theme/theme.ts

```diff
@@ -1,29 +1,27 @@
-import {InjectionToken} from '@angular/core';
-
-export const THEMES = new InjectionToken('THEMES');
-
 export interface Theme {
-  name: string;
-  properties: any;
+  dragenter: {
+    'border': string,
+    'border-radius': string,
+    'border-color': string,
+  };
+  dragover: {
+    'border': string,
+    'border-radius': string,
+    'border-color': string,
+    'background': string,
+  };
```

Comment:
So all of these props are required if the user wants to set custom styles? (I mean what if they want to set only border-radius)